packages/@n8n/nodes-langchain/nodes/agents/Agent/agents/ToolsAgent/execute.ts

```diff
@@ -1,27 +1,27 @@
-import { BINARY_ENCODING, NodeConnectionType, NodeOperationError } from 'n8n-workflow';
-import type { IExecuteFunctions, INodeExecutionData } from 'n8n-workflow';
-
-import type { AgentAction, AgentFinish } from 'langchain/agents';
-import { AgentExecutor, createToolCallingAgent } from 'langchain/agents';
 import type { BaseChatMemory } from '@langchain/community/memory/chat_memory';
+import { HumanMessage } from '@langchain/core/messages';
```

Comment:
Types for the suggestion bellow
```suggestion
import { HumanMessage } from '@langchain/core/messages';
import type { BaseMessage, MessageContentComplex } from '@langchain/core/messages';
```